src/Routing/Filter/LocaleSelectorFilter.php

```diff
@@ -15,6 +15,7 @@
 namespace Cake\Routing\Filter;
 
 use Cake\Event\Event;
+use Cake\I18n\I18n;
```

Comment:
One drawback of this is it creates a depedency on I18n package in case we want to publish Routing as a standalone package in future.